docs/reference/field_types.rst

```diff
@@ -259,3 +259,33 @@ Parameter                   Description
             ])
         ;
     }
+
+Create your own field type
+--------------------------
+
+Field types are Twig templates that are registered in the ``sonata_doctrine_orm_admin`` configuration section.
```

Comment:
IMO, we should keep the docs as much manager agnostic as possible in this package.

Reply:
Let's just deprecate the others managers :p

Reply:
Do you mean that I should move these additions to the docs of SonataDoctrineORMAdminBundle ?

Reply:
It should work the same for every ORM, so I think he just wants you to replace `sonata_doctrine_orm_admin` by something generic like `sonata_doctrine_xxx_admin`.